src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/payment/PayAndSubmitConfirmation.java

```diff
@@ -82,7 +82,7 @@ public PostSubmitCallbackResponse handle(
 
         boolean isException = false;
         try {
-            asylumCase = feePayment.aboutToSubmit(callback);
+            //asylumCase = feePayment.aboutToSubmit(callback);
```

Comment:
@Lee365 not sure what the purpose of this try block is now?

Reply:
This feepayment should not be commented - my mistake.
Please uncomment